src/twisted/internet/base.py

```diff
@@ -473,6 +473,8 @@ class ReactorBase(object):
     @ivar _registerAsIOThread: A flag controlling whether the reactor will
         register the thread it is running in as the I/O thread when it starts.
         If C{True}, registration will be done, otherwise it will not be.
+
+    @ivar exitSignal: The signal that cause the reactor to stop, or 0.
```

Comment:
This attribute appears on some implementations of the reactor now but not others, it's never documented on an interface.